README.md

```diff
@@ -52,6 +52,7 @@ Guides:
  * [Composing Envelopes](Compose-Envelope.md)
 
 Sample apps: 
+ * For SDK version `2.6.0` and above refer our latest: [TGK Financial Sample App](https://github.com/docusign/sample-app-tgk-financial-ios/)
```

Comment:
👍 Thanks for adding reference to new [TGK Financial Sample App](https://github.com/docusign/sample-app-tgk-financial-ios/)